create-uefi-config/main.go

```diff
@@ -24,12 +24,11 @@ import (
 	"encoding/json"
 	"fmt"
 	"io"
-	"io/ioutil"
 	"os"
 	"path/filepath"
 	"sort"
 
-	"github.com/canonical/go-efilib"
+	efi "github.com/canonical/go-efilib"
```

Comment:
Are the `efi` import aliases supposed to be part of this pull request?

Reply:
It's `goimports` doing that automatically for me.

```
❯ goimports -d main.go
diff -u main.go.orig main.go
--- main.go.orig        2025-03-04 10:59:35.939698911 +0100
+++ main.go     2025-03-04 10:59:35.939698911 +0100
@@ -29,7 +29,7 @@
        "strings"
        "time"

-       "github.com/canonical/go-efilib"
+       efi "github.com/canonical/go-efilib"
        "github.com/jessevdk/go-flags"
        log "github.com/sirupsen/logrus"
```

Let me do that in #31 by introducing a dedicated commit.

Reply:
We have another task in our backlog to clean all of this.